README.md

```diff
@@ -153,46 +153,41 @@ into the element. This is the only method that must be implemented by subclasses
   the `render` implementation is a [pure function](https://en.wikipedia.org/wiki/Pure_function) using only the element's current properties
   to describe the element template. This is the only method that must be implemented by subclasses.
   Note, since `render()` is called by `update()`, setting properties does not trigger
-  `invalidate()`, allowing property values to be computed and validated.
+  an update, allowing property values to be computed and validated.
 
   * `shouldUpdate(changedProperties)` (protected): Implement to control if updating and rendering
```

Comment:
I would basically consider basically all of these API's (outside of `render`) "advanced"...